src/components/MainLayout/MainPage/CommonGameArea/GameBoard/GameBoardPart/PanelRow/PanelRow.types.tsx

```diff
@@ -0,0 +1,13 @@
+import { Type } from '../../Player.types';
+
+export enum Warriors {
+  swordsmen,
+  archers,
+  artillery,
+}
+
+export interface PanelRowProps {
+  warriorsType: Warriors;
+  playerType: Type;
+  pwr: number;
```

Comment:
what is pwr?

Reply:
It is power of card. When I throw props in PanelRow, I faced with a limit on the number of characters in a string (100 max) and renamed power to pwr.
Then I rewrote code that calls PanelRow from 1 string to several strings like this:
<PanelRow
prop1=
prop2=
/>
And I forgot to rename pwr to power back. In next commit I will rename pwr to power